### PR TITLE
[CI] Wheel file creation changes for PyPI

### LINKED ISSUE #402: [gpuCI] enable CGA nightly tests
Enable nightly tests for `master` and default `dev-vX.Y.Z` branch in CGA repo.

### PR DESCRIPTION
**Pull request by ohadmo**
Fixes #402

### CODE REVIEW

**Comment by ohadmo** [comment]
@tijyojwad https://stackoverflow.com/a/49609956 solution doesn't seem to actually update the value of the variable, created an issue #443

**Comment by ohadmo** [comment]
@tijyojwad I decided to go with the [development release versioning schem](https://packaging.python.org/guides/distributing-packages-using-setuptools/#choosing-a-versioning-scheme), let me know if you prefer some other option.

According to https://pip.pypa.io/en/stable/reference/pip_install/#pre-release-versions, these pre-release packages are ignored when using `pip install` unless specified explicitly by using a flag.

File: ci/common/test-pyclaragenomics.sh [hunk 1]
```diff
@@ -34,17 +34,26 @@ python setup_pyclaragenomics.py --build_output_folder cga_build
...
-
-cd $PYCLARAGENOMICS_DIR
+cd "$PYCLARAGENOMICS_DIR"
 
 logger "Uninstall pyclaragenomics..."
 pip uninstall -y pyclaragenomics
 
 logger "Create pyclaragenomics Wheel package..."
-python setup_pyclaragenomics.py --build_output_folder cga_build_wheel --create_wheel_only
+CUDA_VERSION=$(cat "$CUDA_HOME"/version.txt | cut -d" " -f3 | cut -d"." -f1-2 | sed -e "s/\./_/g")
```
Comment (tijyojwad) [inline ci/common/test-pyclaragenomics.sh:43]: is this not available in the jenkins environment? I remember seeing this variable in the verbose jenkins output

File: ci/release/pypi_uploader.sh [hunk 2]
```diff
@@ -11,34 +11,30 @@
...
 # Skip upload if CI is executed locally
 if [[ ${RUNNING_CI_LOCALLY} = true  ]]; then
     echo "Skipping PyPi upload - running ci locally"
     return 0
-# Skip upload if CI is not executed locally and current branch is not master
-elif [ "${GIT_BRANCH}" != "${SOURCE_BRANCH}" ]; then
-    echo "Skipping PyPI upload - not master branch"
+fi
+
+# Skip upload if CI is not executed locally and current branch is not master or starts with "dev-"
```
Comment (tijyojwad) [inline ci/release/pypi_uploader.sh:20]: where is the not executed locally condition check?

File: ci/common/test-pyclaragenomics.sh (hunk 1 above)
Comment (ohadmo) [reply ci/common/test-pyclaragenomics.sh]: You're right, thanks!

File: ci/release/pypi_uploader.sh (hunk 2 above)
Comment (ohadmo) [reply ci/release/pypi_uploader.sh]: In the if block above :smiley:

Comment (tijyojwad) [reply ci/release/pypi_uploader.sh]: got, now it's clear! Thanks :)!

**Comment by tijyojwad** [comment]
rerun tests

**Pull request merged**